tests/binaries/makefile

```diff
@@ -0,0 +1,36 @@
+CC              =       gcc
```

Comment:
Most of this file has been taken from gef's tests makefile https://github.com/hugsy/gef/blob/master/tests/binaries/Makefile